### DISCUSSION TITLE
Klappbarer Text und Tabs + Blocks

### DISCUSSION THREAD

**Discussion by Nachtmensch87** [codeispoetry/sunflower#140] [Ideen und Wünsche]
Klappbarer Text existiert ja bereits, Tabs sollen in einem späteren Release kommen.

Ergänzend dazu wäre es super, wenn innerhalb dieser Elemente weitere Blocks genutzt werden können und nicht nur reiner Text.

Siehe folgendes Beispiel (nutzen das grüne Theme JBK): https://gruene-bremen.de/partei/gruen-vor-ort/kreisfreie-stadtteilgruppen/

**Comment by FloJuh** [comment]
Das ist mit dem Details-Block möglich. Es fehlt noch das passende CSS: https://github.com/verdigado/sunflower/issues/549
